packages/salesforcedx-vscode-core/src/commands/util/apexLibraryExecutor.ts

```diff
@@ -81,4 +81,31 @@ export abstract class ApexLibraryExecutor extends LibraryCommandletExecutor<{
       return result;
     };
   }
+
+  public getLogsWrapper(fn: (...args: any[]) => Promise<string[]>) {
+    const commandName = this.executionName;
+
+    return async function(...args: any[]): Promise<string[]> {
+      channelService.showCommandWithTimestamp(`Starting ${commandName}`);
+
+      const result = await vscode.window.withProgress(
+        {
+          title: commandName,
+          location: vscode.ProgressLocation.Notification
+        },
+        async () => {
+          // @ts-ignore
+          return (await fn.call(this, ...args)) as string[];
+        }
+      );
+
+      channelService.showCommandWithTimestamp(`Finished ${commandName}`);
+
+      const logPath = path.join(`${args[0].outputDir}`, `${args[0].logId}.txt`);
```

Comment:
What is the content of this file, the debug log ? If so, it needs to be updated to `.log`